.github/workflows/js_publish.yml

```diff
@@ -0,0 +1,79 @@
+name: Javascript wallet
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Do we really want to trigger this workflow on each PR merge?

Reply:
It will only work after we have establish `tag`